README.md

```diff
@@ -14,3 +15,6 @@ part of the array each recursive call considers.
 Analyse the time complexity of your implementation and give a $\Theta$ bound for
 its worst-case runtime. Add your answer, including your reasoning, to this
 markdown file.
+
+# Answer
+The function starts with base cases which are constant time, $O(1)$. After the base cases the function calculates the length of the input array which is also constant time, $O(n)$. After the base cases and length calculation the function goes into a series of nested loops. The outer loop iterates through size, doubling the value each iteration until size equals the length of the array. The outer loop has $log(n)$ iterations. The inner loop directly inside the outer loop iterates through the elemens in the input array in chunks of size, 2 * size, so the number of iterations is roughly n/size. Within the inner loop elements from the original array are copied to the aux array with a time complexity of $O(size)$. Also in the inner loop, the two subarrays are merged which is also a linear operation so, $O(size)$. Taking the inner loop and outer loop we get a time complexity of $O(n * log(n))$. So the worst case $\Theta$ bound is $\Theta(n * log(n))$. 
```

Comment:
$O(n)$ isn't constant time. And is your merge step really linear? You have to move elements for each merge step, right?